python/ray/rllib/agents/ppo/appo.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import time
+
+from ray.rllib.agents.a3c.a3c_tf_policy_graph import A3CPolicyGraph
+from ray.rllib.agents.ppo.vtrace_policy_graph import VTracePolicyGraph
+from ray.rllib.agents.agent import Agent, with_common_config
+from ray.rllib.optimizers import AsyncSamplesOptimizer
+from ray.rllib.agents import impala
+
+OPTIMIZER_SHARED_CONFIGS = [
+    "lr",
+    "num_envs_per_worker",
+    "num_gpus",
+    "sample_batch_size",
+    "train_batch_size",
+    "replay_buffer_num_slots",
+    "replay_proportion",
+    "num_data_loader_buffers",
+    "max_sample_requests_in_flight_per_worker",
+    "broadcast_interval",
+    "num_sgd_iter",
+    "minibatch_buffer_size",
+]
+
+
+# yapf: disable
+# __sphinx_doc_begin__
+DEFAULT_CONFIG = with_common_config({

```

Comment:
Can you also have this extend the impala config (see apex.py for how to merge dicts).